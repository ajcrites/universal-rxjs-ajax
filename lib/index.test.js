'use strict';

var _rxjs = require('rxjs');

var _index = require('./index');

describe('universal-rxjs-ajax', function () {
  test('request returns Observable', function () {
    var result = (0, _index.request)({ url: 'http://foo.api.af' });
    expect(result instanceof _rxjs.Observable).toBeTruthy();
  });
  test('request returns correct Observable', function () {
    var result = (0, _index.request)({ url: 'http://foo.api.af' });
    expect(result).toMatchSnapshot();
  });
});
//# sourceMappingURL=index.test.js.map