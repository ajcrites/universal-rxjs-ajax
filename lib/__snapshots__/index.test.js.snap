// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`universal-rxjs-ajax request returns correct Observable 1`] = `
AjaxObservable {
  "_isScalar": false,
  "request": Object {
    "async": true,
    "createXHR": [Function],
    "crossDomain": false,
    "headers": Object {},
    "method": "GET",
    "responseType": "json",
    "timeout": 0,
    "url": "http://foo.api.af",
    "withCredentials": false,
  },
}
`;
